packages/chat-component/src/styles/chat-thread-component.ts

```diff
@@ -29,68 +38,82 @@ export const styles = css`
   }
   .chat__footer {
     width: 100%;
-    height: 70px;
+    height: calc(var(--d-large) + var(--d-base));
   }
   .chat__listItem {
-    max-width: 90%;
-    min-width: 80%;
+    max-width: var(--width-wide);
+    min-width: var(--width-base);
     display: flex;
     flex-direction: column;
     height: auto;
 
     @media (min-width: 768px) {
       max-width: 55%;
-      min-width: 50%;
+      min-width: var(--width-narrow);
     }
   }
+  .chat-history__footer {
```

Comment:
this should be in the chat-component and not chat-thread-component

Reply:
Removed. This was likely a merge glitch.

Reply:
Moved.